projects/1.18/assets/materialis/materialis/lang/zh_cn.json

```
@@ -1,195 +1,231 @@
-  "material.materialis.platinum.encyclopedia": "每级额外提升100%耐久消耗速度",
-  "material.materialis.platinum.format": "铂%s",
+  "material.materialis.plastic.encyclopedia": "每级-20%击退",
   "material.materialis.psimetal": "Psi金属",
   "material.materialis.psimetal.encyclopedia": "在工具采掘和攻击的时候施放选定的术式，并增加一个术式插槽",
-  "material.materialis.psimetal.format": "Psi金属%s",
   "material.materialis.quicksilver": "水银",
   "material.materialis.quicksilver.encyclopedia": "工具不再会因为使用而消耗耐久，但每秒有1%的概率消耗耐久",
-  "material.materialis.quicksilver.format": "水银%s",
   "material.materialis.refined_glowstone": "强化荧石",
-  "material.materialis.refined_glowstone.encyclopedia": "采掘速度乘以属性等级除以20得出的值会添加到攻击速度中",
-  "material.materialis.refined_glowstone.format": "强化荧石%s",
+  "material.materialis.refined_glowstone.encyclopedia": "加上采掘速度，乘以效果等级，除以80，等于攻击速度",
   "material.materialis.refined_obsidian": "强化黑曜石",
   "material.materialis.refined_obsidian.encyclopedia": "-20%触及距离",
-  "material.materialis.refined_obsidian.format": "强化黑曜石%s",
   "material.materialis.refined_radiance": "光辉石",
   "material.materialis.refined_radiance.encyclopedia": "对目标造成20秒的发光效果，并在采掘方块后留下光源",
   "material.materialis.refined_radiance.format": "光辉石%s",
+  "material.materialis.regalium": "贵豪",
   "material.materialis.rose_quartz": "玫瑰石英",
   "material.materialis.rose_quartz.encyclopedia": "每个部件提供1个额外的升级槽",
   "material.materialis.rose_quartz.flavor": "免责声明: 实际上并非使用玫瑰制成。",
-  "material.materialis.rose_quartz.format": "玫瑰石英%s",
   "material.materialis.shadow_steel": "暗影钢",
   "material.materialis.shadow_steel.encyclopedia": "击杀的生物和采掘的方块都不再掉落物品，但是击杀生物后会掉落额外的经验值",
-  "material.materialis.shadow_steel.format": "暗影钢%s",
   "material.materialis.starmetal": "星辉",
   "material.materialis.starmetal.encyclopedia": "工具会在午夜获得每级2点的采掘速度",
   "material.materialis.starmetal.flavor": "不要与星形切割金属混淆。",
-  "material.materialis.starmetal.format": "星辉%s",
-  "material.materialis.tungsten": "钨",
-  "material.materialis.tungsten.encyclopedia": "工具的耐久度越低，耐久度的消耗几率越低，最低为1%",
-  "material.materialis.tungsten.format": "钨%s",
+  "material.materialis.terrasteel": "泰拉钢",
+  "material.materialis.terrasteel.encyclopedia": "每级有40%的概率发射一束激光，并消耗魔力（无魔力时则消耗耐久）",
   "material.materialis.uranium": "铀",
-  "material.materialis.uranium.encyclopedia": "耐久度在一半以上的时候会以双倍速度消耗耐久度，在低于一半耐久度的时候耐久度消耗速度则会越来越慢",
-  "material.materialis.uranium.format": "铀%s",
+  "material.materialis.uranium.encyclopedia": "工具在高于四分之一耐久时受到的耐久损耗翻倍，低于四分之一耐久时受到的耐久损耗会越来越少",
   "material.materialis.utherium": "御腐",
   "material.materialis.utherium.encyclopedia": "每级对腐衰生物造成25%的额外伤害",
-  "material.materialis.utherium.format": "御腐%s",
   "modifier.materialis.adrenaline": "肾上腺素",
   "modifier.materialis.adrenaline.description": "生命值越低，攻击伤害越高",
   "modifier.materialis.adrenaline.flavor": "或战或逃！",
   "modifier.materialis.arcane": "奥术",
+  "modifier.materialis.arcane.attack_damage": "魔法伤害",
   "modifier.materialis.arcane.description": "额外对目标造成少量魔法伤害",
   "modifier.materialis.arcane.flavor": "魔法 *哼*",
```

```suggestion
  "modifier.materialis.arcane.flavor": "魔法*嘿嘿*",
```
出处居然是憨豆（）
https://www.youtube.com/watch?v=DM-UnV9TEzY
拟声词应该随便选都行吧